docs/decision-logs-guide/elk/configure.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_label: Configuration
+---
+
+# Configuration
+
+Ingesting aserto decision logs into an analytics platorm, such as [Kibana](https://www.elastic.co/kibana/) and the rest
+of the [ELK stack](https://www.elastic.co/elastic-stack), is a great way of leveraging the value of Aserto. The steps
+described below can get basic ingestion into ELK configured.
+
+## Preliminary Requirements
+- [The Aserto CLI](/docs/command-line-interface/introduction)
+- A tenant id, which can be found in [policy settings](/docs/aserto-console/view-policies#viewing-policy-settings)
+- A [decision logs api key](/docs/decision-logs-guide/console#decision-logs-api-key)
+- A working Elasticsearch environment
+- [Install Logstash](https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html)
+- A working understanding of Logstash
+
+## Use the Aserto CLI to retrieve data
+Assuming a tenant with a tenant id of `0116e83a-7e21-11ec-ab5b-00c9e2c2068b`, create a directory for the logs. For 
+example `~/path/files/decision_logs/0116e83a-7e21-11ec-ab5b-00c9e2c2068b`. Then the command line to retrive new logs
+would be:
+
+```
+aserto --tenant 0116e83a-7e21-11ec-ab5b-00c9e2c2068b decision-logs get \
+--api-key c45cf535d129ddfd0a872b9e5168b7467bcd0c6db9d5160e15c39bfe0784b1a7 \
```

Comment:
should you elide / mask the api key? even if it's a dummy key, it's bad practice to show keys in documentation

Reply:
done